platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/chatter/ChatterCoreTests.java

```diff
@@ -118,24 +118,17 @@ private void loadSavedState(final ChatterCore<GossipEvent> chatterCore, final lo
 
     private List<GossipEvent> generateEventsBelow(final TestingEventBuilder builder, final long lowerBound) {
         final List<GossipEvent> events = new LinkedList<>();
-        events.add(builder.setGeneration(lowerBound - 1).build());
-        events.add(builder.setGeneration(lowerBound - 2).build());
         return events;
     }
 
     private List<GossipEvent> generateEventsAbove(final TestingEventBuilder builder, final long upperBound) {
         final List<GossipEvent> events = new LinkedList<>();
-        events.add(builder.setGeneration(upperBound).build());
-        events.add(builder.setGeneration(upperBound + 1).build());
         return events;
     }
 
     private List<GossipEvent> generateEventsInWindow(
             final TestingEventBuilder builder, final long lowerBound, final long upperBound) {
         final List<GossipEvent> events = new LinkedList<>();
-        events.add(builder.setGeneration(lowerBound).build());
-        events.add(builder.setGeneration((upperBound + lowerBound) / 2).build());
-        events.add(builder.setGeneration(upperBound - 1).build());
         return events;
```

Comment:
these methods now just return empty lists

Reply:
I suggest we don't spend a lot of time maintaining chatter tests at the moment. We will probably have to rewrite these tests from the ground up whenever we end up returning to the chatter family of gossip algorithms.

Reply:
maybe commenting out these lines instead of deleting them would be better

Reply:
fixed